test/tools/libtesteth/TestHelper.cpp

```diff
@@ -452,6 +456,31 @@ void executeTests(const string& _name, const string& _testPathAppendix, const st
 	}
 }
 
+void addClientInfo(json_spirit::mValue& _v, std::string const& _testSource)
+{
+	for (auto& i: _v.get_obj())
+	{
+		json_spirit::mObject& o = i.second.get_obj();
+		json_spirit::mObject clientinfo;
+
+		string head = executeCmd("git rev-parse HEAD");
+		string comment;
+		if (o.count("_info"))
+		{
+			json_spirit::mObject& existingInfo = o["_info"].get_obj();
+			comment = existingInfo["comment"].get_str();
+		}
+
+		//prepare the relative src path
+		string source = _testSource.substr(_testSource.find("/src/"), _testSource.length());
```

Comment:
This does not work with my setup.  My test fillers are in `~/src/tests/src/GeneralStateTestsFiller/stLogTests`.  The first `/src/` is something different from what's sought here.